client/admin/apps/AppsTable.js

```diff
@@ -0,0 +1,125 @@
+import React, { useMemo, useCallback, useState, useEffect } from 'react';
+import { Box, Table, TextInput, Icon, Tag } from '@rocket.chat/fuselage';
+import { useDebouncedValue } from '@rocket.chat/fuselage-hooks';
+
+import AppAvatar from '../../components/basic/avatar/AppAvatar';
+import AppStatus from './AppStatus';
+import { GenericTable, Th } from '../../components/GenericTable';
+import { useLoggedInCloud } from './hooks/useLoggedInCloud';
+import { useTranslation } from '../../contexts/TranslationContext';
+import { useRoute } from '../../contexts/RouterContext';
+import { useFilteredLocalApps } from './hooks/useFilteredLocalApps';
+import { useResizeInlineBreakpoint } from '../../hooks/useResizeInlineBreakpoint';
+import AppMenu from './AppMenu';
+
+const FilterByText = React.memo(({ setFilter, ...props }) => {
```

Comment:
I think we have a o lot of them... should we reuse? maybe create under `GenericTable`

Reply:
Yes definitely. This is part of a conversation we had a few days ago, that we would need to replace this for most tables. But I think it should be done on a separate PR so we can unify all of 'em.